calendar-bundle/src/Resources/contao/classes/Calendar.php

```diff
@@ -353,7 +353,7 @@ public function getSearchablePages($arrPages, $intRoot=0, $blnIsSitemap=false)
 					}
 
 					// The target page has not been published (see #5520)
-					if (!$objParent->published || ($objParent->start != '' && $objParent->start > $time) || ($objParent->stop != '' && $objParent->stop <= $time))
+					if (!$objParent->published || ($objParent->start != '' && $objParent->start > $time) || ($objParent->stop != '' && $objParent->stop < $time))
 					{
```

Comment:
Changing `<=` to `<` changes the behavior. We should not do that IMO because the behavior would be different from the SQL queries then.

Reply:
The positive check needs to be `start <= time` and `stop > time`, the inverse check is `start > time` and `stop <= time`.

Reply:
Reverted in a85dd2b2efc69b49be51fb7f5ebde9974f536140.